flutter_vlc_player_platform_interface/lib/src/utils/options/vlc_subtitle_options.dart

```diff
@@ -0,0 +1,90 @@
+class VlcSubtitleOptions {
+  final List<String> options;
+
+  VlcSubtitleOptions(this.options);
+
+  /// Set subtitle font (must be accessible by vlc)
```

Comment:
Perhaps we could add some documentation on how to make the font available to the native lib?

Reply:
we can but i really don't know how it works.
i tried to set the subtitle font in vlc but it throws the font not found.
the only options i tested are font size, color, thickness & opacity.

Reply:
Yeah, this looks like a tricky one.

I've tried a variety of things:
1. Loading a font to filesystem and pointing the lib to its location.
2. Using a system font by its name (e.g. 'Roboto', 'AndroidClock').
3. Using a system font by its filesystem path ('`/system/fonts/AndroidClock.ttf`').
4. Loading a font via `google_fonts` package and using them by name.
5. Bundling fonts with and iOS app.

None of the approaches seemed to work. I'm not sure we want to keep these options exposed, unless we find a way to get them to actually work.

Reply:
I've also tested the following:
1. Font size, color, opacity, boldness.
2. Background color, opacity.
3. Text direction.
4. Outline opacity, color, thickness.
5. Shadow color, distance and angle.

Looks like the library handles the out-of-range input clipping by itself, so nonsensical inputs behave as if they're clipped to the closest constraint ((opacity `-200.0` -> `0.0`)).

Reply:
did you test that with an external .srt file?!

Reply:
Yeah, I tested with an `.srt`. For built-in subs the font color does not change, but the outline color option works OK.

Reply:
yea, you're right.